File: Programming - 2/Lab_9/Symbols/StringContainer.cs
using System;

namespace Symbols
{
    public delegate int SomeDelegate(char c);
    public class StringContainer
    {
        private static string classLine = "kukosiki";
        private string instanceLine;
        public StringContainer(string line)
        {
            instanceLine = line;
        }
        public static int StaticCharCount(char searchChar)
        {
            int count = 0;
            foreach(char c in classLine)
            {
                if (c == searchChar)
                    count++;
            }
            return count;
        }
        public int InstanceCharCount(char searchChar)
        {
            int count = 0;
            foreach (char c in instanceLine)
            {
                if (c == searchChar)
                    count++;
            }
            return count;
        }
    }
}

File: Programming - 2/Lab_9/Lab_9_CPP/StringContainer.cpp
#include "StringContainer.h"
string StringContainer::classLine = "kukosiki";

StringContainer::StringContainer(string line)
{
	instanceLine = line;
}

int StringContainer::StaticCharCount(char searchChar)
{
    int count = 0;
    for (int i = 0; i < classLine.length(); i++ )
    {
        if (classLine[i] == searchChar)
            count++;
    }
    return count;
}

int StringContainer::InstanceCharCount(char searchChar)
{
    int count = 0;
    for (int i = 0; i < instanceLine.length(); i++)
    {
        if (instanceLine[i] == searchChar)
            count++;
    }
    return count;
}